data/mods/Dark-Skies-Above/factions.json

```
@@ -2,38 +2,47 @@
   {
     "type": "MONSTER_FACTION",
     "name": "invader_alien",
-    "neutral": [ "small_animal", "fish", "animal", "lizard" ]
+    "base_faction": "nether",
+    "neutral": [ "alien_carnivore", "alien_herbivore", "stray", "aquatic_predator" ],
+    "by_mood": "animal"
   },
   {
     "type": "MONSTER_FACTION",
     "name": "alien_herbivore",
-    "base_faction": "small_animal"
+    "base_faction": "herbivore",
+    "neutral": [ "invader_alien", "stray" ],
+    "by_mood": [ "alien_carnivore" ]
   },
   {
     "type": "MONSTER_FACTION",
-    "name": "lizard",
-    "neutral": [ "small_animal", "invader_alien" ],
-    "by_mood": "lizard"
+    "name": "alien_carnivore",
+    "base_faction": "mutant",
+    "neutral": [ "invader_alien", "stray", "aquatic_predator" ],
+    "hate": [ "alien_herbivore" ]
```

```suggestion
    "by_mood": "alien_carnivore",
    "hate": [ "alien_herbivore" ]
```
The bare faction "alien_carnivore" should be used when it's a solitary monster that's perfectly willing to attack others of its species. With the above, factions derived from alien_carnivore will by default be "by_mood" vs other factions derived from it (and the "alien_packliz" `"by_mood": "alien_carnivore"` will not be needed).